packages/components/src/components/common/tabs/tabs.tsx

```diff
@@ -28,4 +41,60 @@ const TabsContent = ({ className, ...props }: React.ComponentProps<typeof TabsPr
 );
 TabsContent.displayName = TabsPrimitive.Content.displayName;
 
-export { Tabs, TabsList, TabsTrigger, TabsContent };
+export type InscriptionTabProps = {
+  id: string;
+  name: string;
+  state: StateDotProps;
+  content: React.ReactNode;
+  icon: IvyIcons;
+};
+
+export type BasicInscriptionTabsProps = {
+  tabs: InscriptionTabProps[];
+  value?: string;
+  onChange?: (change: string) => void;
+};
+
+const BasicInscriptionTabs = ({ tabs, onChange, value }: BasicInscriptionTabsProps) => {
+  const defaultTab = tabs.length > 0 ? tabs[0].id : '';
+  return (
+    <Tabs className={cn(inscriptionTabsRoot, 'ui-inscription-tabs')} defaultValue={defaultTab} value={value} onValueChange={onChange}>
+      <TabsList className={cn(inscriptionTabsList, 'ui-inscription-tabs-list', `tabs-${tabs.length >= 5 ? 'many' : 'few'}`)}>
+        {tabs.map((tab, index) => {
+          const state = tab.state.messages?.find(message => message.variant === 'error')
+            ? 'error'
+            : tab.state.messages?.find(message => message.variant === 'warning')
+              ? 'warning'
+              : undefined;
+
+          return (
+            <TabsTrigger
+              key={`${index}-${tab.id}`}
+              className={cn(inscriptionTabsTrigger, 'ui-inscription-tabs-trigger')}
+              data-message={state}
+              value={tab.name}
+              aria-label={tab.name}
```

Comment:
only the title and the aria-label should be the tab.name, not the value. otherwise the value gets the translated name